examples/monitor-with-thanos/thanos-query.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/component: query-layer
+    app.kubernetes.io/instance: thanos-query
+    app.kubernetes.io/name: thanos-query
+    app.kubernetes.io/version: v0.17.2
+  name: thanos-query
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: query-layer
+      app.kubernetes.io/instance: thanos-query
+      app.kubernetes.io/name: thanos-query
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: query-layer
+        app.kubernetes.io/instance: thanos-query
+        app.kubernetes.io/name: thanos-query
+        app.kubernetes.io/version: v0.17.2
+    spec:
+      containers:
+        - args:
+            - query
+            - --grpc-address=0.0.0.0:10901
+            - --http-address=0.0.0.0:9090
+            - --log.level=debug
+            - --log.format=logfmt
+            - --query.replica-label=prometheus_replica
+            - --query.replica-label=rule_replica
+            - --store=basic-prometheus:10901
+            - --query.timeout=5m
+            - --query.lookback-delta=15m
+            - |-
+              --tracing.config="config":
+                "sampler_param": 2
+                "sampler_type": "ratelimiting"
+                "service_name": "thanos-query"
+              "type": "JAEGER"
```

Comment:
```suggestion
                "type": "JAEGER"
```

Reply:
Reference : https://github.com/thanos-io/kube-thanos/pull/191 .This blank character is correct.